api_tests/utils.py

```diff
@@ -15,3 +15,30 @@ def create_test_file(node, user):
         'contentType': 'img/png'
     }).save()
     return test_file
+
+
+def create_test_file_for_size_filter(node, user):
```

Comment:
@brianjgeiger I did some search within our existing codebase and no FileFactory exists for OsfStorageFile and I kept digging to try to figure out why. It seems that a root node of a folder exists as the root for all OsfFileStorageNodes. Making a factory would only generate an instance of OsfStorageFile. However for the tests to work, we need to append the instance to the root node. The existing "append_file()" method of the root node only takes in a filename as an argument, therefore creating an instance with such filename and append it to the root node. And there is no other method that would take a File instance as an argument and append it to the root node. Creating such a method would mean adding an extra method to the codebase which would only be used for the unit tests.
The best way I can think of is moving the methods in the api_tests/utils.py to the test file itself and make those them private methods for the test classes. I checked and discovered such solution is consistent with other test files regarding osfstorage.